monero-outreach-docs/translations/pt-br/storyboard/blockchain_gender_pt_br.md

```diff
@@ -0,0 +1,78 @@
+# Disparidade de gênero no blockchain: Problema ou potencial?
+
+Monica Mizzi | 12/09/18 Uma entrevista com @sunchakr da Monero Talk.
+
+Discussões e debates sobre blockchain são frequentemente voltados para a análise de aspectos técnicos dessa tecnologia revolucionária, mas **existe um tópico não técnico que sempre chama a atenção: mulheres no blockchain.**
+
+Esse tópico tem provocado aplausos e vaias tanto de mulheres quanto de homens. Embora seja difícil negar a disparidade de gênero nessa indústria, o que [não carece de estatísticas](https://cointelegraph.com/news/women-in-blockchain-and-crypto-how-to-tackle-gender-inequality) que destaquem o menor número de mulheres investindo e trabalhando na indústria, as opiniões das pessoas sobre o assunto ainda são muito divergentes.
+
+De um lado estão aqueles que acreditam que abordar diretamente a disparidade de gênero no blockchain - e na indústria de tecnologia em geral - são essenciais para ajudar a alavancar o status e a participação das mulheres nessa indústria. Do outro lado, estão aqueles que não acreditam que se quer exista um problema de representação.
+
+**"Há tanto o que fazer e há tanta coisa que você pode fazer"**
+[@sunchakr](https://twitter.com/sunchakr)
+
+Entre esses extremos, estão aqueles que reconhecem a necessidade de as mulheres serem melhor representadas e apoiadas, mas acreditam que destacar as estatísticas como as mencionadas acima, enfatiza demasiadamente a negatividade do assunto. Eles argumentam que, embora essa abordagem seja bem intencionada, ela, ironicamente, piora a situação. Eles alegam que quanto mais as mulheres ouvirem sobre os problemas que as mulheres enfrentam nessa indústria, mais elas serão inibidas de entrar ou permanecer nela. Desta forma, este grupo propõe expor as conquistas das mulheres na indústria e focar no progresso que está sendo feito todos os dias.

```

Comment:
Evaluate if we can also remove "na indústria" here: "propõe expor as conquistas das mulheres _na indústria_ e focar no progresso"